src/App.js

```diff
@@ -1,23 +1,33 @@
-import React from 'react';
+import React from "react";
 
-import { getAll, getById } from './api/phone'
-import Basket from './Basket'
-import Filter from './Filter'
-import Catalog from './Catalog'
-
-import './App.css';
+import { getAll, getById } from "./api/phone";
+import Basket from "./Basket";
+import Filter from "./Filter";
+import Catalog from "./Catalog";
 
+import "./App.css";
+// import { isTerminatorless } from "@babel/types";
 
 class App extends React.Component {
-  constructor(props) {
-    super(props);
+  state = {
+    phones: getAll(),
+    selectedPhone: null,
+    basketItems: []
+  };
 
-    this.state = {
-      phones: getAll(),
-      selectedPhone: null,
-      basketItems: [],
-    };
-  }
+  addToCart = (name, id) => {
+    this.setState(prevState => ({
+      basketItems: [...prevState.basketItems, { name, id }]
+    }));
+  };
+
+  removeFromCart = name => {
+    this.setState({
+      basketItems: this.state.basketItems.filter(item =>
+        item.name !== name ? true : false
```

Comment:
```suggestion
        item.name !== name
```
Ternary is redundant. This expression already returns boolean